spread.yaml

```diff
@@ -168,6 +168,20 @@ backends:
                   storage: 20G
                   workers: 1
 
+    qemu-nested:
+        type: qemu
+        memory: 8G
```

Comment:
Perhaps 8GB is too much, I would try with less, even we could use less memory for the nested vm when the backend is qemu. Currently we use 4GB for the nested because we have 8 GB in the gce machines but now for qemu I think it could be less.

Reply:
sure I need to add some additional code to make qemu-nested work actually, so I can change the nested VM's memory to 2G, and then qemu-nested backend memory to 4G (i.e. host VM -> 4G, nested VM -> 2G). Is that okay?